lib/common/web-view-methods.js

```diff
@@ -54,6 +53,7 @@ exports.syncMethods = new Set([
 ])
 
 exports.asyncMethods = new Set([
+  'loadURL',
```

Comment:
Shouldn't there be a corresponding doc change to note the loadURL is now async?

Reply:
@jkleinsc The `<webview>.loadURL` was actually asynchronous while it was under put `syncMethods`, the `WebContents.loadURL` API works asynchronously that it only starts the loading and returns immediately without waiting for anything.

So with this PR `<webview>.loadURL` won't change any documented behavior.

The side effect of this PR is making `<webview>.loadURL` return a `Promise`, but since we are backporting it to 6.0, I think it is probably better keeping it undocumented otherwise it would be a `feat`.